### DISCUSSION TITLE
From Live Preview to edit stylesheet

### DISCUSSION THREAD

**Discussion by simonclay** [brackets-cont/brackets#274] [Q&A]
Hi, I am new to Brackets (from Espresso) and am liking Brackets very much.

I love the way I can click on an element in Live Preview and the html code for that element is then highlighted in code.

**I wonder if there is a feature or plugin that will do a similar thing for CSS?**

ie, if I click on an element in Live Preview, a list appears of the rules in the stylesheet, I then click on one of the rules listed and am taken to that line in the stylesheet.

Here's how it worked in Espresso:
![Screenshot 2023-04-18 at 13 27 57](https://user-images.githubusercontent.com/2143351/232781281-2921da34-793c-409c-9515-ba9e057dfac4.png)

Is there anything like this for Brackets?

Thanks?

**Comment by abose** [comment]
Quick edit is the brackets way of doing it in code. But I see value in a more interactive live preview in the future.

 https://youtu.be/y99hzrAVIZE

**Reply by simonclay** [reply]
That is great @abose. Thank you for bringing that to my attention. 

I agree that more interaction in Live Preview would be good, but Quick Edit is helpful for now. 

Thanks again.
